var/spack/repos/builtin/packages/intel-mkl/package.py

```diff
@@ -69,15 +78,27 @@ def blas_libs(self):
 
         mkl_threading = ['libmkl_sequential']
 
+        omp_libs = LibrayList()
+
         if '+openmp' in spec:
             if '%intel' in spec:
-                mkl_threading = ['libmkl_intel_thread', 'libiomp5']
-            else:
+                mkl_threading = ['libmkl_intel_thread']
+                omp_threading = ['libiomp5']
+
+                omp_root = prefix.compilers_and_libraries.linux.lib.intel64
+                omp_libs = find_libraries(
+                    omp_threading, root=omp_root, shared=shared)
+            elif '%gcc' in spec:
                 mkl_threading = ['libmkl_gnu_thread']
 
+                gcc = Executable(self.compiler.cc)
+                omp_libs = gcc('--print-file-name', 'libgomp.{0}'.format(
```

Comment:
do you mind calling this different from `omp_libs` which is a `LibraryList`? Maybe `path_to_libgomp`. It probably does not matter in Python, but just reads better, IMO